templates/org/team/new.tmpl

```diff
@@ -78,7 +78,7 @@
 										<tr>
 											<th>{{.locale.Tr "units.unit"}}</th>
 											<th class="center aligned">{{.locale.Tr "org.teams.none_access"}}
-											<span class="tooltip vm" data-content="{{.locale.Tr "org.teams.none_access_helper"}}">{{svg "octicon-question" 16 "ml-2"}}</th>
+											<span class="tooltip vm" data-content="{{.locale.Tr "org.teams.none_access_helper"}}">{{svg "octicon-question" 16 "ml-2"}}</span></th>
 											<th class="center aligned">{{.locale.Tr "org.teams.read_access"}}
```

Comment:
Huh, was this HTML code valid before?

Reply:
well, closing tag is not a problem for many browsers...

Reply:
Yup, all modern browsers work well for the lost closing tags.

ps: there are some `<input></input>` later (from the old code), maybe the `</input>` could be removed together because the `input` is void-element and doesn't have closing tag

Reply:
input closes removed